functions/shared_libs/api_clients/zoom_client.py

```
@@ -56,78 +120,509 @@ def fetch_access_token(self):
+                "falling back to report-by-id behaviour."
             )
-        zoom_attendances = self.format_zoom_attendances(zoom_session["participants"])
-        return zoom_attendances
+            participants = self.get_meeting_participants_by_id(meeting_id)
+            return self.aggregate_participants(
+                [dict(p, _meeting_uuid=None, _instance_start=None) for p in participants]
+            )
+
+        try:
+            instances = self.list_past_meeting_instances(meeting_id)
+        except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)